tests/e2e/support/objects/app-files/share/actions.ts

```diff
@@ -60,21 +66,44 @@ export const createShare = async (args: createShareArgs): Promise<void> => {
 
 /**/
 
-export type ShareStatusArgs = Omit<ShareArgs, 'recipients'>
+export interface acceptShareArgs extends ShareArgs {
+  via?: 'STATUS' | 'ACTIONS'
+}
+export type ShareStatusArgs = Omit<acceptShareArgs, 'recipients'>
 
 export const acceptShare = async (args: ShareStatusArgs): Promise<void> => {
-  const { resource, page } = args
-  await Promise.all([
-    page.locator(util.format(shareAcceptDeclineButton, resource, 'status-accept')).click(),
-    page.waitForResponse((resp) => resp.url().includes('shares') && resp.status() === 200),
-    page.locator(util.format(filesSharedWithMeAccepted, resource)).waitFor()
-  ])
+  const { resource, via, page } = args
+  if (via === 'ACTIONS') {
+    await Promise.all([
+      page.locator(util.format(actionMenuDropdownButton, resource)).click(),
+      page.waitForResponse((resp) => resp.url().includes('shares') && resp.status() === 200),
+      page.locator(util.format(actionsTriggerButton, resource, 'accept-share')).click(),
+      page.waitForResponse((resp) => resp.url().includes('shares') && resp.status() === 200),
+      page.locator(util.format(filesSharedWithMeAccepted, resource)).waitFor()
+    ])
+  } else {
+    await Promise.all([
+      page.locator(util.format(shareAcceptDeclineButton, resource, 'status-accept')).click(),
+      page.waitForResponse((resp) => resp.url().includes('shares') && resp.status() === 200),
+      page.locator(util.format(filesSharedWithMeAccepted, resource)).waitFor()
+    ])
```

Comment:
maybe create functions for these and reuse

Reply:
:+1: